ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
+The Foundation suggests that to become an xGov an Algo holder **MUST**:
+- Participate in another reward-bearing AF program
+- Commit the rewards to an xGov DAO for a period
+ 
+The foundation will maintain lists of eligible reward-bearing programs, and a list of qualified DAOs.
+ 
+#### Example
+Alice participates in governance during Q1 2023
+the potential reward is 5 Algo, Alice:
+1) Commit 100 Algo in an AF program
+2) Express her will to participate in xGov
+3) Will not receive 5 Algo at the end of Q1 2023. They are instead locked in one big xGov pool for 1 year
+4) Will receive 5 Algo at Q1 2024
+ 
+## Rationale
+In this ARC we seek community input on issues such as, but not limited to:
+- The types of reward-bearing activities that **SHOULD** be included in the qualification list (ie. general governance)
+- Whether xGovs **MUST** vote on all measures to keep their status
```

Comment:
Yes. Voting on all measures should be required, but there should be an option to abstain from voting. xGovs should be aware of all measures, but if the xGov is not qualified to vote on the measure, then the xGov should have the option to abstain from the vote.

Reply:
@oysterpack This makes sense. Since these are pretty meta "votes on votes" I assume they will have a simple "support" / "not support" binary decision so it will be simple to abstain.